daemon/src/maker_cfd.rs

```diff
@@ -467,122 +469,80 @@ where
         // Since rejection is tied to the cfd state at the moment we can only do this after creating
         // a cfd.
         if quantity < current_order.min_quantity || quantity > current_order.max_quantity {
```

Comment:
What do you think of moving this check also into the `setup_maker` actor?

Reply:
It felt weird to spawn an actor to let it "fail" immediately, but I guess the `maker_cfd` actor shouldn't even be involved in all this anyway.

Reply:
https://github.com/Restioson/xtra/blob/224004bc95dac12b8b8d2a14fea5ddf37eef6ae0/src/context.rs#L194-L200 :man_shrugging:

Reply:
Hahaha, I feel seen.